components/filter-components/MapFilters.vue

```diff
@@ -670,6 +677,17 @@ export default {
         el.scrollIntoView({ behavior: "smooth" });
       }
     },
+    getGlossaryLabel: function (label) {
+      if (label === 'Gender' || label === 'Shapeshifting' || label === 'Wikipedia Page' ){
+        return null
+      }
+      if (label === 'Primary' || label === 'Secondary' ){
+        return 'Case Characterisations'
+      }
+      else {
```

Comment:
We don't need the else. I think we can just have a standalone `return label` at the end of the function